e2e/test/scenarios/question/summarization.cy.spec.js

```diff
@@ -174,19 +174,22 @@ describe("scenarios > question > summarize sidebar", () => {
     cy.findByText("318.7");
   });
 
-  it.skip("should keep manually entered parenthesis intact (metabase#13306)", () => {
+  it("should keep manually entered parenthesis intact (metabase#13306)", () => {
     const FORMULA =
       "Sum([Total]) / (Sum([Product → Price]) * Average([Quantity]))";
 
     openOrdersTable({ mode: "notebook" });
     summarize({ mode: "notebook" });
+
     popover().contains("Custom Expression").click();
     popover().within(() => {
-      cy.get(".ace_text-input").type(FORMULA).blur();
+      enterCustomColumnDetails({ formula: FORMULA });
+      cy.get("@formula").blur();
+
+      cy.wait(100); // waits for formula update after blur
```

Comment:
We could rely on some visible effect of updating the formula rather than just waiting for some arbitrary amount. One idea is to rely on `SUM([Total])` being renamed to `Sum([Total])`.